test/component/Legend.spec.tsx

```diff
@@ -2480,7 +2480,6 @@ describe('<Legend />', () => {
       const { container } = render(
         <RadarChart width={500} height={500} data={numericalData}>
           <Legend />
-          {/* @ts-expect-error TypeScript correctly points out that dataKey is required but I want a test for this anyway */}
```

Comment:
no longer required?

Reply:
Not sure why. I will change Radar props later so I will take a look.